package.json

```diff
@@ -2,17 +2,18 @@
   "name": "open-ai-reviewer",
   "version": "1.0.0",
   "description": "Open AI powered code reviews",
-  "main": "lib/main.js",
+  "main": "lib/index.js",
   "author": "Ville Saukkonen",
   "license": "MIT",
   "scripts": {
-    "build": "tsc",
-    "package": "ncc build --source-map --license licenses.txt",
+    "build": "tsc --sourceMap",
```

Comment:
The `--sourceMap` option should be checked for compatibility with your TypeScript configuration.